camunda-formio-tasklist-vue/.eslintrc.js

```diff
@@ -12,21 +12,25 @@ module.exports = {
     ecmaVersion: 2020,
   },
   rules: {
-    "no-console": "warn",
+    "no-console": process.env.NODE_ENV === "production" ? "warn" : "off",
     "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
-    "vue/no-unused-components": "warn",
-    "sort-imports": "warn",
-    camelcase: "warn",
+    "vue/no-unused-components": "error",
+    "camelcase": "warn",
     "vue/valid-template-root": "warn",
-    indent: ["warn", 2],
-    eqeqeq: "warn",
+    "indent": ["warn", 2],
+    "eqeqeq": "warn",
     "handle-callback-err": "warn",
     "prefer-promise-reject-errors": "warn",
     "no-unused-vars": "warn",
     "no-undef": "warn",
     "@typescript-eslint/no-inferrable-types": "off",
     "@typescript-eslint/no-non-null-assertion": "off",
-    "space-before-function-paren": ["warn", "always"],
     "semi": ["warn", "always"],
+    "eol-last": ["error", "always"],
+    "block-spacing": "error",
+    "linebreak-style": "error",

```

Comment:
![image](https://user-images.githubusercontent.com/70306694/133041850-ee69f357-28f3-48db-a070-826cd1395fd1.png)

The linebreak-style was giving an error, when I was running in windows for testing. If users of project are using linux and windows follows the style differently. Won't it cause conflicts?